live-examples/js-examples/promise/promise-constructor.html

```diff
@@ -1,9 +1,23 @@
 <pre>
 <code id="static-js">var promise1 = new Promise(function(resolve, reject) {
-  setTimeout(resolve, 100, 'foo');
+  setTimeout(function() { 
```

Comment:
I think it's weird to mix arrow functions and non-arrow functions, is there a reason you're doing this?

Reply:
Let me know if I need to do any other changes. Thanks for your time.